etc/mirror-box.ts

```diff
@@ -16,7 +17,25 @@ export async function mirror(name: string) {
   });
 }
 
+export async function mirrorDatabase() {
+  const start = process.hrtime();
+  const spinner = ora(`${chalk.yellowBright("Mirroring Database")}`).start();
+  spinner.start(`${chalk.yellowBright("Fetching next data...")}`);
+
+  fetchDatabase()
+    .then(() => {
+      const end = `${toSecond(process.hrtime(start))} seconds`;
+      spinner.succeed(
+        `Finished mirroring Database in ${chalk.greenBright(end)}`,
+      );
+    })
+    .catch((err) => {
+      chalk.red(err);
+    });
+}
+
```

Comment:
Please exclude this file from the code coverage collection just like what we did in #788, thanks!